src/Writers/HTMLWriter.jl

```diff
@@ -740,6 +746,8 @@ function mdconvert(link::Markdown.Link, parent)
         path = first(s)
         path = endswith(path, ".md") ? Formats.extension(:html, path) : path
         url = (length(s) > 1) ? "$path#$(last(s))" : Compat.String(path)
+        # Replace any backslashes in links, if building the docs on Windows
+        url = replace(url, '\\', '/')
```

Comment:
I meant the person writing the Markdown files and docstrings. Any generated URLs should be sanitized already. But tracking this down could be a bit of work, so it would be fine to keep this here as it is, and figure this detail out later.